deploy/olm-catalog/service-binding-operator/0.0.20/service-binding-operator.v0.0.20.clusterserviceversion.yaml

```diff
@@ -0,0 +1,153 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
@Avni-Sharma What is the value of this file in this repo? Is it used for generating a fresh one before pushing?

Reply:
Yes

Reply:
Could you add a readme to the relevant directory indicating its purpose - could be confused with the actual version out there otherwise.